composer.json

```diff
@@ -17,7 +17,8 @@
   "require": {
     "php": ">=7.1",
     "symfony/yaml": "^2.8|^3",
-    "zendframework/zend-filter": "^2.0.0"
+    "zendframework/zend-filter": "^2.0.0",
+    "symfony/process": "^3.0.0"
```

Comment:
Please make sure the order is correct.
Also please check phpstan, we should try to use 0.10.

Reply:
Fixed the order. Checked 0.10 for phpstan - got 6 errors, skip for now.